script/imbrogno_export.sql

```diff
@@ -0,0 +1,7 @@
+-- imbrogno_export and the SQL-Query imbrogno_export.sql:
+-- Exports all datasets into a csv file, where the location is within a radius of 3000 meter from
+-- the citycenter of: Uppsala, Hong Kong, Seoul, Taipei, Amsterdam, Perth, Sydney, Honolulu and Munich
+-- to prevent a timeout of the API
+SET search_path TO "$user", public, topology, postgis;
```

Comment:
Is there a reason we're setting search_path here but not in the daily file?

Reply:
It was just there so I don't need it to write this every time. Thought that this statement is not needed in the final version, so I put the statements out of the code. But unfortunately I forgot this one.